time.go

```diff
@@ -11,18 +11,18 @@ const (
 	defaultRTT = 1 * time.Second
 )
 
+func makeTime(t int64) time.Time {
+	return time.Unix(t/1000, (t%1000)*int64(time.Millisecond))
+}
+
 func timestamp(t time.Time) int64 {
 	if t.IsZero() {
 		return 0
 	}
 	return t.UnixNano() / int64(time.Millisecond)
 }
 
-func timestampToTime(t int64) time.Time {
-	return time.Unix(t/1000, (t%1000)*int64(time.Millisecond))
-}
-
-func duration(ms int32) time.Duration {
+func makeDuration(ms int32) time.Duration {
```

Comment:
I renamed `duration` to `makeDuration` to unify the naming pattern of a few unexported functions.